content/6-network-security/4-fully-private-cluster/1-create-cluster/index.en.md

```diff
@@ -406,44 +427,43 @@ VPC endpoint for EC2 messages
 
 ```bash
 aws ec2 create-vpc-endpoint \
-    --vpc-id $CLUSTER_VPC \
+    --vpc-id $PRIVATE_CLUSTER_VPC \
     --vpc-endpoint-type Interface \
     --service-name com.amazonaws.$AWS_REGION.ec2messages \
-    --subnet-ids $AZ1_SUBNET $AZ2_SUBNET \
+    --subnet-ids $PRIVATE_AZ1_SUBNET $PRIVATE_AZ2_SUBNET $PRIVATE_AZ3_SUBNET \
     --security-group-id $VPCE_SG \
     --tag-specifications 'ResourceType=vpc-endpoint,Tags=[{Key=service,Value=ec2messages}]'
 ```
 
-
-
 ```bash
 aws eks describe-cluster --name eksworkshop-eksctl-private --query cluster.endpoint
 ```
 
-
 This will display the API endpoint of the cluster.
 
-```
-"https://AFB4045AF25413FF766AD8CA1FF0CAEA.yl4.ap-south-1.eks.amazonaws.com"
-```
+::::expand{header="Check Output"}
 
-Confirm that the API endpoint is private by using the nslookup command. You can see the private IP addressed assigned from the VPC CIDR range.
+"https://AFB4045AF25413FF766AD8CA1FF0CAEA.yl4.us-west-2.eks.amazonaws.com"
 
-```bash
-nslookup AFB4045AF25413FF766AD8CA1FF0CAEA.yl4.ap-south-1.eks.amazonaws.com
-```
+::::
 
-````
-Server:         172.31.0.2
-Address:        172.31.0.2#53
+Confirm that the API endpoint is private using the nslookup command. You can see the private IP addresses returned for the control plane endpoint is assigned from the New Private EKS cluster VPC CIDR range.
+
+::::expand{header="Check Output"}
```

Comment:
This is example output, it miss the command to run to have it ?

Reply:
added